Add example command to create /etc/apt/keyrings directory (#43626)
kubernetes/website: content/en/docs/tasks/tools/install-kubectl-linux.md

2. Download the public signing key for the Kubernetes package repositories. The same signing key is used for all repositories so you can disregard the version in the URL:

   ```shell

>    # If the folder `/etc/apt/keyrings` does not exist, it should be created before the curl command, read the note below.

Reviewer: ```suggestion
   # If the directory /etc/apt/keyrings does not exist, you should create it (see the note below).
```

   # sudo mkdir -p -m 755 /etc/apt/keyrings
   curl -fsSL https://pkgs.k8s.io/core:/stable:/{{< param "version" >}}/deb/Release.key | sudo gpg --dearmor -o /etc/apt/keyrings/kubernetes-apt-keyring.gpg
   ```
   
{{< note >}}

> In releases older than Debian 12 and Ubuntu 22.04, folder `/etc/apt/keyrings` does not exist by default, and it should be created before the curl command.

Reviewer: ```suggestion
In releases older than Debian 12 and Ubuntu 22.04, the directory `/etc/apt/keyrings` does not exist by
default; if you don't have this directory, create it first.
```
